.github/workflows/claude.yml

```
@@ -36,28 +39,28 @@ jobs:
           # This is an optional setting that allows Claude to read CI results on PRs
           additional_permissions: |
             actions: read
-
+          
           # Optional: Specify model (defaults to Claude Sonnet 4, uncomment for Claude Opus 4)
-          model: "claude-opus-4-20250514"
-
+            model: "claude-opus-4-20250514"
+          
           # Optional: Customize the trigger phrase (default: @claude)
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)